domains/stoltz.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "stoltz-dev",
+        "email": "rafaelbraun5@hotmail.com"
+    },
+    "record": {
+        "URL": "stoltz.is-a.dev"
```

Comment:
Invalid URL record (missing protocol)
also you would cause a redirect loop with those records